multihost_test/bz_automation/test_automation.py

```diff
@@ -66,3 +66,42 @@ def test_read_faillock_conf_option(self, multihost, create_localusers):
                                "local_anuj --reset")
         execute_cmd(multihost, "cp -vf /etc/security/faillock.conf_bkp "
                                "/etc/security/faillock.conf")
+
+    def test_cve_2010_3316(self, multihost):
+        """
+        :title: CVE-2010-3316-pam_xauth-missing-return-value-checks-from-setuid
+        :id: aebe751c-31b8-11ed-a91f-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=637898
+        """
+        TUSER = "pam-xauth-tester"
+        SLOG = "/var/log/secure"
+        execute_cmd(multihost, "cp -vf /etc/pam.d/su /etc/pam.d/su_anuj")
+        execute_cmd(multihost, "cp -vf /etc/security/limits.conf "
+                               "/etc/security/limits.conf_anuj")
```

Comment:
You could use the [bkp_pam_config()](https://github.com/RedHatQE/pam/blob/main/multihost_test/bz_automation/conftest.py#L61) to take care of the backup and restore.

Reply:
DOne